index.js

```diff
@@ -1113,12 +1113,20 @@
 
   //# equals :: (a, b) -> Boolean
   //.
-  //. Returns `true` if its arguments are of the same type and equal according
-  //. to the type's [`fantasy-land/equals`][] method; `false` otherwise.
+  //. Returns `true` if its arguments are equal; `false` otherwise.
   //.
-  //. `fantasy-land/equals` implementations are provided for the following
-  //. built-in types: Null, Undefined, Boolean, Number, Date, RegExp, String,
-  //. Array, Arguments, Error, Object, and Function.
+  //. Specifically:
+  //.
+  //.   - Arguments of different types are unequal.
```

Comment:
Should we mention that we're using sanctuary-type-identifiers to differentiate types?

Reply:
Yes, that's a good idea.

Reply:
How about *Arguments with different [type identities](https://github.com/sanctuary-js/sanctuary-type-identifiers) are unequal*?

Reply:
Excellent! We can use `v:sanctuary-js/sanctuary-type-identifiers` to link to the right version. :)